kratos/integration/integration_info.h

```diff
@@ -0,0 +1,238 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+
+#if !defined(KRATOS_INTEGRATION_INFO_H_INCLUDED )
+#define  KRATOS_INTEGRATION_INFO_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "geometries/geometry_data.h"
+#include "containers/data_value_container.h"
+#include "containers/flags.h"
+
+namespace Kratos
+{
+
+///@name Kratos Classes
+///@{
+
+/// Integration information for the creation of integration points.
+/* Within this class distinct information of integration can be
+ * stored and processed.
+ */
+class IntegrationInfo : public Flags

```

Comment:
```suggestion
class KRATOS_API(KRATOS_CORE) IntegrationInfo : public Flags
```

This should fix win errors.

Reply:
eval windows. thanks a lot for the fix!